operator/Makefile

```
@@ -198,7 +198,7 @@ manifests: parent-proto-generate controller-gen ## Generate WebhookConfiguration
 	$(CONTROLLER_GEN) rbac:roleName=manager-role crd webhook paths="./..." output:crd:artifacts:config=config/crd/bases
 
 .PHONY: generate
-generate: parent-proto-generate controller-gen ## Generate code containing DeepCopy, DeepCopyInto, and DeepCopyObject method implementations.
+generate: parent-proto-generate controller-gen mapkubeapis ## Generate code containing DeepCopy, DeepCopyInto, and DeepCopyObject method implementations.
```

Can this be removed? 🤔 

```suggestion
generate: parent-proto-generate controller-gen ## Generate code containing DeepCopy, DeepCopyInto, and DeepCopyObject method implementations.
```

---

Thanks, removed